WordPress/Classes/Utility/WPThemeSettings.m

```diff
@@ -0,0 +1,71 @@
+#import "WPThemeSettings.h"
+
+static NSString *WPThemeSettingsUserDefaultsKey = @"WPThemeSettingsUserDefaultsKey";
+static NSString *WPThemeSettingsURLScheme = @"themes";
+static NSString *WPThemeSettingsURLQueryTurnOff = @"enabled=0";
+static NSString *WPThemeSettingsURLQueryTurnOn = @"enabled=1";
+
+@implementation WPThemeSettings
+
+#pragma mark - URL handling
+
++ (BOOL)handleURL:(NSURL *)url
+{
+    BOOL result = NO;
+    NSString *query = [url query];
+    
+    if ([query isEqualToString:WPThemeSettingsURLQueryTurnOn]) {
+        [self enable];
+        result = YES;
+    } else if ([query isEqualToString:WPThemeSettingsURLQueryTurnOff]) {
```

Comment:
Might be nice to make disabling the feature more flexible so that any query not matching `WPThemeSettingsURLQueryTurnOn` would disable the feature.